eden/converter/RNA/RNA_PLFOLD.py

```diff
@@ -0,0 +1,80 @@
+import networkx as nx
+import subprocess as sp
+from eden.modifier.FASTA import FASTA
+import os
+
+def RNAplfold_wrapper(sequence, **options):
+    # default settings.
+    path_to_program = options.get('path_to_program', '/usr/local/vrna/2.1.8/bin/RNAplfold') # Error reported if full RNAplfold path not given
```

Comment:
I would always assume that RNAfold is on your $PATH so you can remove this entirely.